### DISCUSSION TITLE
읽는속도 질문

### DISCUSSION THREAD

**Discussion by catpooll** [Lastorder-DC/chatreader-kor#44] [문의]
느리게 바꿀수있는방법 없을까요

**Comment by summercatkr** [comment]
!!tts uniq off  치면 보통 TTS 속도로 바뀝니다.

**Comment by Lastorder-DC** [comment]
uniq off 하시면 기본속도로 돌아갑니다. 아직 개별 보이스 속도를 조절할수는 없는데 추가 예정입니다.
